dfusion_rust_core/src/models/order.rs

```diff
@@ -25,6 +26,27 @@ pub struct Order {
     pub sell_amount: u128,
 }
 
+impl Order {
+    pub fn from_encoded_order (
+        account_id: u16,
+        bytes: Vec<u8>
```

Comment:
Could this be a fixed length slice also? We should only ever be able to convert vectors with 26 elements and it would be nice if the compiler can enforce that. 
I guess it might be a problem converting a chunk into a fixed size slice, but maybe we can do that in the call-site with some form of unwrap.